app/assets/stylesheets/admin/views/_editions-index.scss

```diff
@@ -21,7 +21,7 @@
     }
   }
 
-  .label.force_published {
+  .document-label {
     margin-left: 1em;
```

Comment:
Could we see how this looks if the document has multiple pills?

Reply:
Just added a screenshot